examples/cusparselt/matmul.py

```diff
@@ -6,8 +6,8 @@
 import cupy
 import numpy
 
+from cupy_backends.cuda.api import runtime
```

Comment:
Maybe `from cupy.cuda import runtime` is better? I don't think we documented `cupy_backends`?

Reply:
Fixed!